src/minisweagent/environments/docker.py

```diff
@@ -89,7 +89,7 @@ def execute(self, command: str, cwd: str = "") -> dict[str, Any]:
         result = subprocess.run(
             cmd,
             text=True,
-            timeout=self.config.timeout,
+            timeout=timeout if timeout is not None else self.config.timeout,
```

Comment:
All of these can be written in a more concise way as `timeout or self.config.timeout`